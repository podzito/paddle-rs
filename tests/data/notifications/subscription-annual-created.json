{
  "data": {
    "address_id": "add_01j5g2nvcrec9av2mewb1q0k62",
    "billing_cycle": { "frequency": 1, "interval": "year" },
    "billing_details": null,
    "business_id": null,
    "canceled_at": null,
    "collection_mode": "automatic",
    "created_at": "2024-08-17T11:49:00.14Z",
    "currency_code": "USD",
    "current_billing_period": { "ends_at": "2024-08-24T11:49:00.129Z", "starts_at": "2024-08-17T11:49:00.129Z" },
    "custom_data": null,
    "customer_id": "ctm_01j535fawggst1gm3qsyanc0mb",
    "discount": null,
    "first_billed_at": null,
    "id": "sub_01j5g2pq3cwtx43fd2h40c67e9",
    "import_meta": null,
    "items": [
      {
        "created_at": "2024-08-17T11:49:00.14Z",
        "next_billed_at": "2024-08-24T11:49:00.129Z",
        "previously_billed_at": null,
        "price": {
          "billing_cycle": { "frequency": 1, "interval": "year" },
          "created_at": "2024-08-12T09:44:03.962458Z",
          "custom_data": { "credits": "25", "storage": "5000" },
          "description": "PodZito Pro - Annual",
          "id": "pri_01j52zjbkt405xwe2hdx1rpz00",
          "import_meta": null,
          "name": "PodZito Pro",
          "product_id": "pro_01j52z87e5sdt3p60mmtcavpgt",
          "quantity": { "maximum": 1, "minimum": 1 },
          "seller_id": "21860",
          "status": "active",
          "tax_mode": "account_setting",
          "trial_period": { "frequency": 7, "interval": "day" },
          "type": "standard",
          "unit_price": { "amount": "19999", "currency_code": "USD" },
          "unit_price_overrides": [],
          "updated_at": "2024-08-13T17:52:27.702555Z"
        },
        "product": {
          "created_at": "2024-08-12T09:38:32.005Z",
          "custom_data": null,
          "description": "Podzito",
          "id": "pro_01j52z87e5sdt3p60mmtcavpgt",
          "image_url": null,
          "import_meta": null,
          "name": "PodZito",
          "seller_id": "21860",
          "status": "active",
          "tax_category": "standard",
          "type": "standard",
          "updated_at": "2024-08-12T09:38:32.005Z"
        },
        "quantity": 1,
        "recurring": true,
        "status": "trialing",
        "trial_dates": { "ends_at": "2024-08-24T11:49:00.129Z", "starts_at": "2024-08-17T11:49:00.129Z" },
        "updated_at": "2024-08-17T11:49:00.14Z"
      }
    ],
    "next_billed_at": "2024-08-24T11:49:00.129Z",
    "paused_at": null,
    "scheduled_change": null,
    "started_at": "2024-08-17T11:49:00.129Z",
    "status": "trialing",
    "transaction_id": "txn_01j5g2nqc36pyfk7rhymdfw10j",
    "updated_at": "2024-08-17T11:49:00.14Z"
  },
  "event_id": "evt_01j5g2pr0m1a2ykkh7cs5v2jgp",
  "event_type": "subscription.created",
  "notification_id": "ntf_01j5g2prembgkmxefenzny5aft",
  "occurred_at": "2024-08-17T11:49:01.076460Z"
}
