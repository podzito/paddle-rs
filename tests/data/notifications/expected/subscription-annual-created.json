{
  "event_id": "evt_01j6ffeybs8663frsz3240kjch",
  "notification_id": "ntf_01j6ffeyg71gse6bm6fp8pvacr",
  "event": {
    "SubscriptionCreated": {
      "id": "sub_01j6ffexwgztae4bsmnxn1h2zs",
      "customer_id": "ctm_01j535fawggst1gm3qsyanc0mb",
      "billing_cycle": { "frequency": 1, "interval": "year" },
      "next_billed_at": "2025-08-29T16:28:22.856323Z",
      "paused_at": null,
      "scheduled_change": null,
      "started_at": "2024-08-29T16:28:22.856323Z",
      "custom_data": null,
      "items": [
        {
          "price": {
            "billing_cycle": { "frequency": 1, "interval": "year" },
            "created_at": "2024-08-29T11:38:23.207406Z",
            "custom_data": null,
            "description": "PodZito Pro Plus",
            "id": "pri_01j6eyvx3707phydse7hf79ksp",
            "import_meta": null,
            "name": "PodZito Pro Plus",
            "product_id": "pro_01j52z87e5sdt3p60mmtcavpgt",
            "quantity": { "maximum": 1, "minimum": 1 },
            "status": "active",
            "tax_mode": "account_setting",
            "trial_period": null,
            "type": "standard",
            "unit_price": { "amount": "12999", "currency_code": "USD" },
            "updated_at": "2024-08-29T11:38:23.207406Z"
          },
          "product": {
            "created_at": "2024-08-12T09:38:32.005Z",
            "custom_data": null,
            "description": "Podzito",
            "id": "pro_01j52z87e5sdt3p60mmtcavpgt",
            "image_url": null,
            "name": "PodZito",
            "status": "active",
            "tax_category": "standard",
            "type": "standard",
            "updated_at": "2024-08-12T09:38:32.005Z"
          },
          "created_at": "2024-08-29T16:28:23.824Z",
          "next_billed_at": "2025-08-29T16:28:22.856323Z",
          "previously_billed_at": "2024-08-29T16:28:22.856323Z",
          "price_id": null,
          "quantity": 1,
          "recurring": true,
          "status": "active",
          "trial_dates": null,
          "updated_at": "2024-08-29T16:28:23.824Z"
        }
      ],
      "status": "active",
      "transaction_id": "txn_01j6ffdrbabr6a8zyrdsjz28ev",
      "updated_at": "2024-08-29T16:28:23.824Z"
    }
  },
  "occurred_at": "2024-08-29T16:28:24.313864Z"
}
